spec/menufactory_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'menufactory'
+require 'menu'
+require 'dish'
+
+describe MenuFactory do
+
+  let(:list) { [{name: :sushi, price: 5},
+                {name: :burger, price: 3},
+                {name: :noodles, price: 4}]
+    }
```

Comment:
Lint/BlockAlignment: `end` at 10, 4 is not aligned with `let(:list) { [{name: :sushi, price: 5},` at 7, 2.